x-pack/test_serverless/functional/test_suites/search/config.ts

```diff
@@ -14,4 +17,8 @@ export default createTestConfig({
     reportName: 'Serverless Search Functional Tests',
   },
   suiteTags: { exclude: ['skipSvlSearch'] },
+  serverArgs: findTestPluginPaths([
```

Comment:
I decided to only load/support example plugins for the Search project since it seemed redundant to run them for all project types. Alternatively we could add this to `x-pack/test_serverless/functional/config.base.ts` which would load them for all project types, and put the example plugin tests in the `common` folder, although it would mean they'd run three times in CI (one for each project type).

Reply:
I guess this is not set for Search projects on MKI? If so, we won't be able to run it in real environment on regular basis, unless there is a way to make it work without re-defining Kibana server configuration. Ideally we would like to be able to run functional tests on both local/Kibana CI and real serverless environments, what are your Team expectations?

Reply:
Thanks for the feedback @dmlemeshko!

>I guess this is not set for Search projects on MKI?

I don't know much MKI to be honest, but I'd assume not since these are example plugins not intended to run in prod.

>If so, we won't be able to run it in real environment on regular basis, unless there is a way to make it work without re-defining Kibana server configuration.

Oops... that's quite a drawback then 😬 I'm not aware of a way this can be done without using the `--plugin-path` arg since it seems to be how we test additional plugins everywhere in Kibana.

>Ideally we would like to be able to run functional tests on both local/Kibana CI and real Serverless environments, what are your Team expectations?

These are the same expectations we have and how I thought it would work, but I'm now seeing I was confused 🙂 The tests would still be useful running against local/CI Serverless since they could catch any Serverless specific client side changes that affect the tested components, but not nearly as useful as in a real Serverless environment of course, and not what I was intending with this PR.

I'm assuming this means there's no way to customize the server args when running tests on MKI and my tests would just fail, right? And if there isn't another way to load example plugins, none of the example plugin tests can be migrated to Serverless?

Reply:
@dmlemeshko do you have advice on how the team can get this to work?